utils/service_discovery/abstract_config_store.py

```diff
@@ -26,7 +26,8 @@
 INIT_CONFIGS = 'init_configs'
 INSTANCES = 'instances'
 KUBE_ANNOTATIONS = 'kube_annotations'
-KUBE_ANNOTATION_PREFIX = 'com.datadoghq.sd/'
+KUBE_CONTAINER_NAME = 'kube_container_name'
+KUBE_ANNOTATION_PREFIX = 'sd.datadoghq.com'
```

Comment:
So I may have snuck this one in there. I changed the annotation to be `service-discovery.datadoghq.com/checks.XXXX`. I've realized that kube annotations don't follow the "reverse dns" scheme at all and I didn't want to deviate from that. Let me know if you'd prefer to change it back to what you were suggesting.